config/default/eu_cookie_compliance.settings.yml

```diff
@@ -54,9 +54,9 @@ cookie_name: ''
 exclude_uid_1: false
 better_support_for_screen_readers: false
 method: categories
-disabled_javascripts: 'google_analytics:modules/contrib/localgov_eu_cookie_compliance/js/deferred-ga-script-runner.js'
+disabled_javascripts: "analytics_cookies:modules/contrib/localgov_eu_cookie_compliance/js/deferred-ga-script-runner.js\r\nanalytics_cookies:modules/contrib/localgov_eu_cookie_compliance/js/deferred-hotjar-script-runner.js"
 automatic_cookies_removal: true
-allowed_cookies: "analytics_cookies:_ga*\r\nanalytics_cookies:_ga"
+allowed_cookies: "analytics_cookies:_ga*\r\nanalytics_cookies:_ga\r\nanalytics_cookies:_hj*\r\nanalytics_cookies:NEXT_LOCALE\r\nanalytics_cookies:ps_mode\r\nanalytics_cookies:Optanon*\r\nanalytics_cookies:_BEAMER*\r\nanalytics_cookies:_gcl_au\r\nanalytics_cookies:acq_homepage*"
```

Comment:
Hi @siliconmeadow 

Some of these are surprising to me. The page at https://help.hotjar.com/hc/en-us/articles/6952777582999-Cookies-Set-by-the-Hotjar-Tracking-Code shows only cookies starting `_hj*` and `hj*`. What are `NEXT_LOCALE`, `ps_mode`, `Optanon*`, `_BEAMER`, `_gcl_au` and `acq_homepage*`?

Reply:
While testing hotjar, I set up actions and found those extra cookies being set:

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/158008/232877097-97d8e3c5-ede8-4076-9d7a-396c69c7d92e.png">

They're mentioned here as functional cookies:

https://help.hotjar.com/hc/en-us/articles/115011789248

Reply:
Sorry. I see the err of my ways re: 3rd party cookies.